<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showdata</class>
 <widget class="QWidget" name="showdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton::hover
{
background-color:#93b4ff;
}
QPushButton::pressed
{
background-color:#93b4ff;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/img/返回箭头.png);
color: rgb(255, 0, 0);
borde: none;
}
QPushButton::hover
{
background-color:#93b4ff;
}
QPushButton::pressed
{
background-color:#93b4ff;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string>      数   据   展   示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/pm2.5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/紫外线 (2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="roomteLb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="uaLb">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 16pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32)</string>
   </property>
   <property name="text">
    <string>紫外线（mW/㎡）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32)</string>
   </property>
   <property name="text">
    <string> PM2.5(μg/m3）</string>
   </property>
  </widget>
  <widget class="QLabel" name="kitchenLb">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>264</y>
     <width>601</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bathLb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/客厅2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/卧室icon新.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/厨房 (4).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/浴室.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32);
</string>
   </property>
   <property name="text">
    <string>当前温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32)</string>
   </property>
   <property name="text">
    <string>当前湿度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/温度.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/湿度.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32)</string>
   </property>
   <property name="text">
    <string>是否有人：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/温度.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32);
</string>
   </property>
   <property name="text">
    <string>当前温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/湿度.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32)</string>
   </property>
   <property name="text">
    <string>当前湿度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/火.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32);
</string>
   </property>
   <property name="text">
    <string>当前火光状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/烟雾传感器.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32);
</string>
   </property>
   <property name="text">
    <string>当前烟雾状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/人体红外开.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/二氧化碳.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/甲烷.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32)</string>
   </property>
   <property name="text">
    <string>CO2(PPM）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32)</string>
   </property>
   <property name="text">
    <string>CH4（PPM）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/报警器.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/报警灯.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32);
</string>
   </property>
   <property name="text">
    <string>当前报警器状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(	218 ,165, 32);
</string>
   </property>
   <property name="text">
    <string>当前报警灯状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ch4Lb">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 16pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="co2Lb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 16pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pmLb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 16pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="roomhumLb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="livingteLb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="livinghumLb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="peopelLb">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="winLb">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fireLb">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frogLb">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sosLb">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sosledLb">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timeLb">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
